group_project_v2/notifications.py

```diff
@@ -195,9 +202,14 @@ def fire_grades_posted_notification(self, group_id, notifications_service):
         location = unicode(self.location) if self.location else ''
         add_click_link_params(msg, unicode(self.course_id), location)
 
-        # Bulk publish to the 'group_project_workgroup' user scope
-        notifications_service.bulk_publish_notification_to_scope(
-            NotificationScopes.WORKGROUP,
-            {'workgroup_id': group_id},
-            msg
+        send_at_date_tz = send_at_date.replace(tzinfo=pytz.UTC)
```

Comment:
@ziafazal I've blown up my devstack , so it would take time to actually verify it, but it looks like send_at_date might be None (i.e. if there are no GradeDisplayStages or they are all always open - have open_date set to None).

I would just add `send_at_date = send_at_date if send_at_date is not None else datetime.now()` above this line.